services/src/main/java/org/keycloak/theme/DefaultThemeManager.java

```diff
@@ -106,20 +115,19 @@ private Theme loadTheme(String name, Theme.Type type) {
         List<Theme> themes = new LinkedList<>();
         themes.add(theme);
 
-        if (theme.getImportName() != null) {
-            String[] s = theme.getImportName().split("/");
-            themes.add(findTheme(s[1], Theme.Type.valueOf(s[0].toUpperCase())));
-        }
+        if (!processImportedTheme(themes, theme, name, type)) return null;
```

Comment:
Shouldn't this return the default theme?

Reply:
Actually not as `loadTheme` is private method, which is supposed to return the theme itself just if it exists and if it is OK. Even before this PR, the method returned null in case that theme itself is not found. So I've just extended this to return null also in case that "parent" is null or "import" is null (which means theme was found, but is incorrect).

The fallback to default theme is already done in the calling method https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/theme/DefaultThemeManager.java#L68-L72 .